### DISCUSSION TITLE
Confused on the modulo bit

### DISCUSSION THREAD

**Discussion by Epic-Fafnir** [GDQuest/learn-gdscript#467] [Q&A]
Explanation on the dice generator lesson why does it say max_number+1 confused on why that's there what it's supposed to do I know it limits the Randi function but how and the process of why. P.S The app has been really great stop using it for a while cause of exams and my laptop had to get fixed, really beginner-friendly as I am a beginner and allows you to learn at a great pace.

**Comment by UbeJelly** [comment]
Let's say the max_number is equal to 5, by using randi() you'll have 0, 1, 2, 3, 4 numbers.

By adding 1, that is _max_number + 1_, each individual number would increase by 1:
from _0, 1, 2, 3, 4_ now you have _1, 2, 3, 4, 5_.

Here's another example:
![randi()_test](https://user-images.githubusercontent.com/69304294/173185148-0241e21d-d28e-4d53-857d-9f4bdeeb1ccc.png)
